### ISSUE TITLE
Icon set request: meter-0, meter-1, meter-2, meter-3, meter-4

### ISSUE THREAD

**Issue by DoktorJ** [FortAwesome/Font-Awesome#7884]
It'd be nice to have "meter" icons; something more generic than the existing battery icons. This could represent capacity _or_ speed (e.g. your storage space is X full, or the process is running at X speed). I've made some SVGs as a representative sample. If you'd like, I can provide the SVGs to use as is or modify as you see fit :)

**Comment by DoktorJ** [comment]
Here are smaller, more icon-sized exports of my SVGs. In the second one, I brought the marks in closer to the needle and tightened their spread (25° apart vs 30° in the first run).

![meters](https://cloud.githubusercontent.com/assets/13531701/11162080/33f595fc-8a5f-11e5-8b4b-dc6bf7f9a3f5.png)
![meters-2](https://cloud.githubusercontent.com/assets/13531701/11162081/33f7c570-8a5f-11e5-89f4-67df00360d76.png)

**Comment by DoktorJ** [comment]
What feedback is being waited for? Most other open issues with this tag are waiting on input from the requester, but I see no inquiry for feedback here.

**Comment by tagliala** [comment]
@DoktorJ "waiting for feedback" is a label I'm 'using for multiple purposes, this is not only related to feedback from OP 

If I well recall, this is a duplicate. I'm taking my time to investigate.

Hope it helps

**Comment by dpfem** [comment]
+1

**Comment by wwilliams** [comment]
:+1:

**Comment by DoktorJ** [comment]
@tagliala any thought on this? As I indicated, there is a single static "gauge" icon (fa-tachometer -- which isn't useful as a progress/capacity indicator). There are also a couple of other indicators, but they are more specific; there's a battery indicator and an hourglass, but neither are appropriate for something such as speed or disk capacity (the battery _sort of_ works for disk capacity, but it's not ideal).

**Comment by tagliala** [comment]
since this does not appear to be a duplicate, please fix the title and I will add proper labels

It should reflect the exact number of icons you are requesting and match battery's naming (e.g: tachometer-0, tachometer-1...)

**Comment by DoktorJ** [comment]
Changed the title. Aliases should be meter-empty, meter-quarter, meter-half, meter-three-quarters, meter-full, as with battery.

**Comment by sensibleworld** [comment]
We have a few "gauge" icons that might fit the bill for this: https://fontawesome.com/v6.0/icons?d=gallery&p=1&q=gauge&s=solid

**Comment by DoktorJ** [comment]
> We have a few "gauge" icons that might fit the bill for this: https://fontawesome.com/v6.0/icons?d=gallery&p=1&q=gauge&s=solid

Thanks for adding these!
